install/installer/traf_add_sudoAccess

```diff
@@ -23,12 +23,12 @@
 TRAF_CONFIG=/etc/trafodion/trafodion_config
 source $TRAF_CONFIG
 
-if [ $node_count -eq 1 ]; then
+if [ $all_node_count -eq 1 ]; then
     TRAF_PDSH=""
     TRAF_PDCP=""
 else
-    TRAF_PDSH="pdsh -R exec $MY_NODES $PDSH_SSH_CMD"
-    TRAF_PDCP="pdcp -R ssh $MY_NODES"
+    TRAF_PDSH="pdsh -R exec $ALL_NODES $PDSH_SSH_CMD"
+    TRAF_PDCP="pdcp -R ssh $ALL_NODES"
 fi
```

Comment:
Can generalize the TRAF_PDSH and TRAF_PDCP environment variables in the config file that is sourced in.  Then these conditional checks for setting them can be removed from each script